src/dao/ChronoBankPlatformDAO.js

```diff
@@ -19,7 +19,7 @@ export default class ChronoBankPlatform extends AbstractContractDAO {
 
   async reissueAsset (token, value) {
     try {
-      const amount = tokenService.getDAO(token.id()).addDecimals(value, token)
+      const amount = token.addDecimals(value, token)
```

Comment:
Do not manipulate `value: Amount` in daos!
For correct working `_tx` use `new BigNumber(value: Amount)` below.
Also use `value` for raw values (bigNumbers with decimals) from SC and amount for `Amount` instances for clarity, please.